_posts/2021-09-17-the-joy-of-slow-programming.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: "The joy of slow programming"
```

Comment:
Referencing https://en.wikipedia.org/wiki/Zen_and_the_Art_of_Motorcycle_Maintenance you could call it "Zen and the Art of Side Project Maintenance" :)
Might get you some unexpected clicks :)
(But I totally get it if you want "slow programming" in the title, it's a phrase worth coining!)

Reply:
I read this book 😄 (or at the very least started reading), because I remember the title.

Yes, I'd like to establish the phrase "slow programming", similarly to "slow food" 😉

Reply:
I think "Slow programming" and "boring code" are really good examples of [concept handles](https://notes.andymatuschak.org/z5vA4vw86DKNq22xt6pRWhumeRmSzwV6hxRHE):

> Successful concept handles can really amplify a vague idea which many people sort of understand but can’t point to and talk about. If you give that vague notion a crisp, catchy name, you can unlock a lot of conversation and reflection

Reply:
Just realized your title already IS a pop culture reference, whether intentional or not: to "The Joy of Painting" with Bob Ross :smile:

Reply:
I didn’t do that consciously, but the phrase “the joy of X” just sounded good to me 😅

Reply:
"Concept handle" is a concept handle too :repeat: :slightly_smiling_face: